spec/index.bs

```diff
@@ -52,6 +52,94 @@ a {{GPURenderPipeline}} or a {{GPUComputePipeline}} object. The state not includ
 in these [=pipeline=] objects is set during encoding with commands,
 such as {{GPUCommandEncoder/beginRenderPass()}} or {{GPURenderPassEncoder/setBlendColor()}}.
 
+Security considerations {#security}
+=====================
+
+## CPU undefined driver behavior ## {#security-cpu-ub}
```

Comment:
```suggestion
## CPU-undefined driver behavior ## {#security-cpu-ub}
```

Reply:
It doesn't sound right: "CPU-undefined" reads to me as "undefined by CPU" where in fact it's undefined by the spec, not CPU.

Reply:
Ohhh, I see. How about "CPU-based" and "GPU-based undefined driver behavior"?